kcidb/oo/__init__.py

```diff
@@ -581,6 +582,42 @@ def valid(self):
         )
         return self.__getattr__("valid") if valid is None else valid
 
+    @cached_property
+    def get_build_failure_log(self):
+        """Get one-liner build error from log_excerpt.
+        Return None if self.log_excerpt is not set"""
```

Comment:
This is not the only case where `None` could be returned, it will also be returned if `log_excerpt` is empty or consist entirely of whitespace, which could be good to mention.

Reply:
Fixed it.